components/sidebar/features.h

```diff
@@ -0,0 +1,22 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_SIDEBAR_FEATURES_H_
+#define BRAVE_COMPONENTS_SIDEBAR_FEATURES_H_
+
+#include "base/feature_list.h"
+#include "base/metrics/field_trial_params.h"
+
+namespace sidebar::features {
+
+// Whether to show the sidebar always on stable channel.
+BASE_DECLARE_FEATURE(kSidebarShowAlwaysOnStable);
+
+// Whether to open the leo panel only once.
+extern const base::FeatureParam<bool> kOpenOneShotLeoPanel;
```

Comment:
nit: for consistency

```suggestion
extern const base::FeatureParam<bool> kOpenOneTimeLeoPanel;
```

Reply:
Updated to use `OneShot` all places as it's more widely used :)